spec/models/manageiq/providers/kubernetes/container_manager/metrics_capture/capture_context_spec.rb

```diff
@@ -0,0 +1,96 @@
+describe ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::CaptureContext do
+  before(:each) do
+    allow(MiqServer).to receive(:my_zone).and_return("default")
+    # Recording new cassetes will require to set up a new sample enviorment
+    # with a provider and hawkular metrics
+    hostname = 'yzamir-centos7-1.eng.lab.tlv.example.com'
+    hostname_hawkular = 'yzamir-centos7-1.eng.lab.tlv.example.com'
+    token = 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ' \
```

Comment:
@yaacov this token should remain local only and not be committed (used only for recording)